app/controllers/candidate_interface/course_choices_controller.rb

```diff
@@ -83,6 +88,10 @@ def pick_course
       end
     end
 
+    def full
+      @course = Course.find_by(id: params[:course_id], provider_id: params[:provider_id])
```

Comment:
I think we don't need the provider param here? Then this can be reduced to `Course.find(params[:course_id])`.

Reply:
🤦‍♂ updated!